src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -69,18 +40,19 @@
 import com.ericsson.eiffel.semantics.events.EiffelActivityFinishedEvent;
```

Comment:
please modify the events package also import *

Reply:
There are nearly 150 classes in that package and we are using 30 classes in this class is it ok to make it as import * ?